packages/renderer/src/lib/kube/KubernetesDashboardGuideCard.svelte

```diff
@@ -0,0 +1,19 @@
+<script lang="ts">
+import { Button } from '@podman-desktop/ui-svelte';
+
+interface Props {
+  title: string;
+  link: string;
+}
+
+let { title, link }: Props = $props();
+
+function openLink() {
+  window.openExternal(link);
+}
```

Comment:
suggestion (non-blocking): svelte-check is suppose to complain that there is no return type, weird it says nothing

```suggestion
function openLink(): Promise<boolean> {
  return window.openExternal(link);
}
```

Reply:
blocking for me (need return type + await or catch or return)

Reply:
Fixed.